### ISSUE TITLE
Add `coalesceFindRequests` to the default behaviour.

### ISSUE THREAD

**Issue by cibernox** [samselikoff/ember-cli-mirage#121]
Right now having `this.get('/users')` in the config file and doing `GET /users?ids=[1,2,3]` returns all the users in the collection.

It's that possible ATM? if not, is it planned?

**Comment by samselikoff** [comment]
Not possible via shorthands currently, would have to use the function fallback. I could see merging that in with the shorthand, though it might be nice to have an api for that so it's explicit what the shorthand is doing.

**Comment by cibernox** [comment]
I don't know if it's explicit enough, but a configuration option can be enough:

```js
// in /mirage/config.js
this.coalesceFindRequests = true;
```

Alternatively if you don't want that behaviour system-wide:

```js
this.coalesceFindRequests = ['/users', '/posts']
// or
this.get('/users', { coalesce: true });
this.get('/posts', { coalesce: true });
```

**Comment by samselikoff** [comment]
I like the last option. One of my goals for this project is for frontenders to be able to hand their Mirage config.js file to a backend dev and say, "here's the API my app expects." So imo the more explicit the better.

We should think through all of these types of options though and make sure this is the right place for them.

**Comment by cibernox** [comment]
PR coming soon

**Comment by cibernox** [comment]
closed via #123
